packages/configs/typescript/package.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
the other packages don't seem to be using these tsconfigs yet, is that still to do?

Reply:
likewise with vitest

Reply:
Yeah, they can be used already, it's just that they don't require that much configuration, so they're pretty much ready to go. They can be used as soon as they are published.

Reply:
Same with Vitest, they can be used as soon as they're published, I tested them with Yalc, they seem to work just fine.

Reply:
gotcha - i figured through the magic of yarn workspaces they might be able to be used without publishing (though obviously we'll publish them so our other repos can use them)

Reply:
They probably can, I just never got around to figuring that out.